app/build.gradle

```diff
@@ -21,16 +21,16 @@ android {
         all {
             proguardFiles(file('../proguard').listFiles())
             proguardFile getDefaultProguardFile('proguard-android.txt')
-            minifyEnabled true
-            useProguard true
+            //minifyEnabled true TODO
+            //useProguard true
```

Comment:
I can't and will not merge things that are not production ready. This change makes master practically not deployable. Please consider opening a feature branch and merging any changes there, until everything is done.

Reply:
To be clear: we are depending on proguard to reduce APK size and make sure, that we don't reach the max methods (we don't want to use multidex)

Reply:
It’s not my intention to get rid of proguard, obviously. I had some problems during testing, so I deactivated it and forgot to remove the comments.